
Creating a localized database which will characterize NextGen Sequence Reads

I would like to map and characterize somatic mutations that are found in Next Generation Sequence DNA reads using programming and a number of different bioinformatics software.  The mutations would be typed based on nucleic base insertions and deletions and protein coding region.  In addition, the nucleotide position and chromosome location for each mutation would be identified.  These tasks would be carried out first by performing nucleic acid assemblies and alignments with information obtained from a genome browser and using software such as bowtie2 and velvet.  Sequences of interest then could be distinguished from others by utilizing variant differentiation software tools such as SamTools and TopHat. 

To automate the tasks and categorize the data in a manner that would facilitate easy access, the programming would be placed in a pipeline utilizing UNIX scripting and python programming.  Moreover, the data will be organized in a database.  Routine queries into the database would be made more user friendly by creating a GUI front end.
